### PR TITLE
Metrics App

### PR DESCRIPTION
**Pull request by massazoum**
In this project,

-  Metrics web app created and Implemented.
- **Url video** : https://drive.google.com/file/d/14QHW5pC8p4kM3eXzSeySSJrj1yVDiRro/view?usp=sharing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,141 @@
...
+<a name="readme-top"></a>
+### NationsConnect APP USING REACT
+
+# 📗 Table of Contents
+
+- [📖 About the Project]()
+  - [🛠 Built With](#build-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
```
Comment (andersonlebon) [inline README.md:11]: - [ ] Nice work so far however This project should be deployed and accessible online using [Render](https://render.com/), [Netlify](https://www.netlify.com/) or [GitHub Pages](https://pages.github.com/) then add a link to the online version of your app to your README file.

- [ ] Also in your presentation video, Kindly you should also highlight some interesting piece of code or something you built that you are very proud of.
image

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @massazoum 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there 

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by massazoum** [comment]
- I have put the live demo of my NatonConnect site.
- Url video in PR where I added the part explaining my codes.

**Review by Danieldotcomcoder — APPROVED** [review]
Hello @massazoum , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 👍 👍 😄 👏 

Congratulations! 🎉 🎉 🎉 

## Highlights
- Professional Readme File ✔✔
- Linters are setup correctly ✔✔
- Clear and nice code ✔✔

comp-2
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**